src/zenml/integrations/sklearn/__init__.py

```diff
@@ -21,7 +21,7 @@ class SklearnIntegration(Integration):
     """Definition of sklearn integration for ZenML."""
 
     NAME = SKLEARN
-    REQUIREMENTS = ["scikit-learn<1.3"]
+    REQUIREMENTS = ["scikit-learn>1.3"]
```

Comment:
```suggestion
    REQUIREMENTS = ["scikit-learn"]
```

Since we're unblocking / releasing the requirement to versions > 1.3, let's not block the people who are on lower versions.